server/server.py

```diff
@@ -46,25 +82,50 @@ def verify_student(_id):
     Uses the ID that was sent to the student's email to set them as confirmed
     in the database.
     '''
+    logging.info('Hit route %s', request.path)
+    try:
+        obj_id = ObjectId(_id)
+    except Exception:
+        error = jsonify({'error': 'A problem has occurred with the id', 'verified': False})
+        return make_response(error, 400)
+
+    updated = db.update({'_id': obj_id}, {'verified': True})
+
+    if updated:
+        return jsonify({'msg': 'User verified successfully', 'verified': True})

```

Comment:
Do we not need `make_response` here as well??

Reply:
With Flask, when you return a json it creates a response object with it, so if you don't have to add anything more to the response object you can just return the json